hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/StateContext.java

```diff
@@ -149,6 +150,7 @@
    */
   private final AtomicLong heartbeatFrequency = new AtomicLong(2000);
 
+  private final AtomicLong reconHeartbeatFrequency = new AtomicLong(2000);
```

Comment:
nit. Why is the value of this AtomicLong different than the default value of the heartbeat?

Reply:
We keep the value this way to make the initial registration faster. This will be reassigned to the default value during endpoint registration.